README.md

```diff
@@ -1,14 +1,21 @@
-# Azure SQL binding for Azure Functions - Preview
+# Azure SQL bindings for Azure Functions - Preview
 
 [![Build Status](https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_apis/build/status/SQL%20Bindings/SQL%20Bindings%20-%20Nightly?branchName=main)](https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/latest?definitionId=481&branchName=main)
 
 ## Introduction
 
-This repository contains the Azure SQL bindings for Azure Functions extension code as well as a quick start tutorial and samples illustrating how to use the binding in different ways.  A high level explanation of the bindings is provided below. Additional information for each is in their respective sample sections.
+This repository contains the Azure SQL bindings for Azure Functions extension code as well as a quick start tutorial and samples illustrating how to use the binding in different ways. The types of bindings supported are:
 
-- [**Input Binding**](#input-binding-tutorial): takes a SQL query to run and returns the output of the query in the function.
-- [**Output Binding**](#output-binding-tutorial): takes a list of rows and upserts them into the user table (i.e. If a row doesn't already exist, it is added. If it does, it is updated).
-- [**Trigger Binding**](#trigger-binding-tutorial): monitors the user table for changes (i.e., row inserts, updates, and deletes) and invokes the function with updated rows.
+- **Input Binding**: takes a SQL query to run and returns the output of the query in the function.
+- **Output Binding**: takes a list of rows and upserts them into the user table (i.e. If a row doesn't already exist, it is added. If it does, it is updated).
+- **Trigger Binding**: monitors the user table for changes (i.e., row inserts, updates, and deletes) and invokes the function with updated rows.
+
+See the language specific guides for further details on support and usage of these bindings in the supported languages.
+
+- [.NET](./docs/BindingsGuide_Dotnet.md)
+- [Java](./docs/BindingsGuide_Java.md)
+- [Javascript](./docs/BindingsGuide_Javascript.md)
+- [Python](./docs/BindingsGuide_Python.md)
```

Comment:
Can we just condense what we have here with that on line 22 https://github.com/Azure/azure-functions-sql-extension/pull/471/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R22